include/output/sdl_cava.h

```diff
@@ -1,7 +1,7 @@
 #include "../config.h"
 
 void init_sdl_window(int width, int height, int x, int y);
-void init_sdl_surface(int *width, int *height, char *const fg_color_string,
+void init_sdl_surface(int *w, int *h, char *const fg_color_string,
                       char *const bg_color_string, int gradient, int gradient_count,
```

Comment:
Again, looks like these changes I also didn't notice during refactoring. Possible need to rollback once PR is approved to merge